azurerm/resource_arm_iothub_resource_provider_test.go

```diff
@@ -0,0 +1,110 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMIotHub_basicStandard(t *testing.T) {
```

Comment:
_minor_ can we split this into two tests - one `_basic` without the SKU block, and one `_complete` with it?